### PR TITLE
BTA-1752 Currency attribute is now optional

### PR DESCRIPTION
**Pull request by jjromeo**

### CODE REVIEW

File: transaction-flow.md [hunk 1]
```diff
@@ -794,13 +802,17 @@ Funding transactions can be done using the `POST /v1/accounts/debits` endpoint,
 }
 ```
 
+
```
Comment (linniethepooh) [inline transaction-flow.md]: unnecessary spacing

File: transaction-flow.md [hunk 2]
```diff
@@ -794,13 +802,17 @@ Funding transactions can be done using the `POST /v1/accounts/debits` endpoint,
...
-* The `amount` has to be the same as the `input_amount` on the transaction
 * The `to_id` is the `id` of the transaction
 * You need to have enough balance of the appropriate currency inside your wallet.
 
+If you choose to include the optional currency and/or amount params:
+* The `currency` needs to be the same as the `input_currency` on the transaction.
+* The `amount` has to be the same as the `input_amount` on the transaction
+
+
 Once the transaction is funded, we will immediately start trying to pay out the recipient(s).
```
Comment (linniethepooh) [inline transaction-flow.md]: unnecessary spacing

File: transaction-flow.md (hunk 1 above)
Comment (jjromeo) [inline transaction-flow.md]: ``
```suggestion
```

File: transaction-flow.md [hunk 3]
```diff
@@ -796,11 +804,14 @@ Funding transactions can be done using the `POST /v1/accounts/debits` endpoint,
...
-* The `currency` needs to be the same as the `input_currency` on the transaction.
-* The `amount` has to be the same as the `input_amount` on the transaction
 * The `to_id` is the `id` of the transaction
 * You need to have enough balance of the appropriate currency inside your wallet.
 
+If you choose to include the optional currency and/or amount params:
+* The `currency` needs to be the same as the `input_currency` on the transaction.
+* The `amount` has to be the same as the `input_amount` on the transaction
+
+
```
Comment (linniethepooh) [inline transaction-flow.md]: unnecessary spacing

Comment (jjromeo) [inline transaction-flow.md]:
```suggestion
```

File: transaction-flow.md [hunk 4]
```diff
@@ -785,6 +785,14 @@ Once you have an internal account, you can use that as well to fund transactions
 
 Funding transactions can be done using the `POST /v1/accounts/debits` endpoint, with the following body:
 
+```json
+{
+  "to_id": "5f44026b-7904-4c30-87d6-f8972d790ded",
+  "to_type": "Transaction"
+}
+```
+
```
Comment (linniethepooh) [inline transaction-flow.md]: unnecessary spacing

File: transaction-flow.md [hunk 5]
```diff
@@ -785,6 +785,14 @@ Once you have an internal account, you can use that as well to fund transactions
...
+with *optional* currency and amount params
```
Comment (sztupy) [inline transaction-flow.md]:
```suggestion
You can also supply the `currency` and `amount` parameters, in which case we'll verify if they match the amount on the transaction and only fund it if they do
```

File: transaction-flow.md (hunk 4 above)
Comment (jjromeo) [inline transaction-flow.md]:
```suggestion
```

**Pull request merged**